docs/wallet_browser_extension/intro.md

```diff
@@ -0,0 +1,53 @@
+# ParallelChain Wallet Extension
+
+ParallelChain Wallet Extension is an extension for managing your XPLL and integrating your DApp.
+It allows your DApp to interact with your DApp users' XPLL accounts and make smart contract calls through the extension.
+
+# Features
```

Comment:
Put these headings into subheadings, to differentiate them from the heading "ParallelChain Wallet Extension". The formatting now looks a bit untidy.

Reply:
Solved by 0486a15.